source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
-
-The audit test consists of clicking links to an item within a book twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Requests must be recorded. If the two clicks occur with more than 30 seconds between, then 2 Total_Item_Requests must be counted. In both cases, only 1 Unique_Item_Requests and 1 Unique_Title_Requests will be reported.
-
-**Option 1**: Content provider offers OA_Gold items in addition to Controlled items.
-
-The auditor must carry out a total of 32 tests, each test will consist of 2 requests. There are 2 types of tests that must be carried out:
-
-* “Inside” tests (Two requests are made to the same book item, and the second request is made within 30 seconds of the first).
-* “Outside” tests (Two requests are made to the same book item, and the second request is made over 30 seconds after the first).
-
-The auditor must carry out 16 **inside** tests.
-
-* 8 tests to book items where the Access_Type is Controlled and 8 tests to book items where the Access_Type is OA_Gold.
-* Where the site allows, each book must have 2 item tests. This will report 2 Total_Item_Requests, 2 Unique_Item_Requests, and 1 Unique_Title_Requests.
-
-This must result in
-
-* 8 Controlled Total_Item_Requests and 8 OA_Gold Total_Item_Requests in the TR_B3 Standard View.
-* 8 Controlled Unique_Item_Requests and 8 OA_Gold Unique_Item_Requests in the TR_B3 Standard View.
-* 4 Controlled Unique_Title_Requests and 4 OA_Gold Unique_Title_Requests in the TR_B3 Standard View.
-
-The auditor must carry out 16 **outside** tests.
-
-* 8 tests to book items where the Access_Type is Controlled and 8 tests to book items where the Access_Type is OA_Gold.
-* Where the site allows, each book must have 2 item tests. This will report 4 Total_Item_Requests, 2 Unique_Item_Requests, and 1 Unique_Title_Requests.
-
-This must result in:
-
-* 16 Controlled Total_Item_Requests and 16 OA_Gold Total_Item_Requests in the TR_B3 Standard View.
-* 8 Controlled Unique_Item_Requests and 8 OA_Gold Unique_Item_Requests in the TR_B3 Standard View.
-* 4 Controlled Unique_Title_Requests and 4 OA_Gold Unique_Title_Requests in the TR_B3 Standard View.
-
-**Option 2**: Content provider does not offer OA_Gold items.
-
-The auditor must carry out a total of 32 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
-
-* “Inside” tests (Two requests are made to the same book item and the second request is made within 30 seconds of the first).
-* “Outside” tests (Two requests are made to the same book item, and the second request is made over 30 seconds after the first).
+Within the Title Report, breakdowns by Access_Type are essential. There are therefore a series of audit tests designed to determine report providers’ compliance with requirements for reporting Access_Type. These tests only apply where Journals with more than one Access_Type are available on a platform.
```

Comment:
Something seems to be wrong - there are no tests for platforms with a single Access_Type. Either there should be separate Metric_Type tests for journals (as for books) or this section should have an option for platforms with a single Access_Type.

Reply:
added new tests